src/services/review/RepoCheckerService.ts

```diff
@@ -705,8 +739,16 @@ export default class RepoCheckerService {
 
   mediaPathDecoder = (mediaPath: string) => {
     for (let i = 0; i < ALLOWED_FILE_EXTENSIONS.length; i += 1) {
-      if (mediaPath.toLowerCase().endsWith(ALLOWED_FILE_EXTENSIONS[i])) {
-        return decodeURI(mediaPath)
+      let pathWithoutQueryParams = mediaPath
+      if (mediaPath.includes("?")) {
+        pathWithoutQueryParams = mediaPath.split("?").slice(0, -1).join("")
```

Comment:
to be safe here, i think we should take **only the first portion** - if there's 2 `?`, this might give the wrong result (only ignores ending `?`)

Reply:
modified [6a6fc29](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/6a6fc296ed62a9292c47c50ad3dc0917908c8d85)